environment.yml

```diff
@@ -44,6 +43,7 @@ dependencies:
   - coverage
   - flake8>=3.8
   - flake8-docstrings>=1.4.0
+  - fontconfig
```

Comment:
This unskips the following test. Apparently fontconfig does not work.

```
__________________________ test_get_fontconfig_fonts __________________________
[gw0] win32 -- Python 3.8.13 C:\Miniconda3-x64\envs\mpl-dev\python.exe
    @pytest.mark.skipif(not has_fclist, reason='no fontconfig installed')
    def test_get_fontconfig_fonts():
>       assert len(_get_fontconfig_fonts()) > 1
E       assert 0 > 1
E        +  where 0 = len([])
E        +    where [] = _get_fontconfig_fonts()
lib\matplotlib\tests\test_font_manager.py:78: AssertionError
```

Reply:
Eh, not exactly, `fontconfig` is actually a transitive dependency anyway, so that test was failing and I tried _adding_ it explicitly to see if that _fixed_ it (it did not) because the error message indicated that it might be a version thing. Didn't get around to removing it again.

Reply:
fontconfig is brought in in a couple ways with this environment.yml: (and additional ones on unix systems, which made it harder to track down)

but notably, graphviz transitively depends on fontconfig via `pango` and through `libgd` (graphviz directly depends on fontconfig on unix)

Some of the gtk/cairo may also depend on it transitively, though those were mostly `host` dependencies not `run` dependencies from what I saw.

Reply:
Okay, the crux is that `_get_fontconfig_fonts` relies on subprocess call to `fc-config --help` (to determine if a supported version is available)

but the windows version of this utility does not use posix-style options. (At least I think that's why, testing out windows style options now)

Reply:
actually, looking at where `_get_fontconfig_fonts` is called, it doesn't get called on windows, so I just explicitly skipped that test

Reply:
Yes, I don't think there's any point in running this test on Windows, as `fontconfig` is not used there.